draft-ietf-quic-recovery.md

```diff
@@ -816,48 +816,92 @@ The minimum congestion window is the smallest value the congestion window can
 decrease to as a response to loss, ECN-CE, or persistent congestion.
 The RECOMMENDED value is 2 * max_datagram_size.
 
-## Slow Start
+## Congestion Control States
 
-While in slow start, a NewReno sender increases the congestion window by the
-number of bytes acknowledged when each acknowledgment is processed, resulting in
+The congestion controller has three distinct states, as shown in
+{{fig-cc-fsm}}.
+
+~~~
+                 Start or         +------------+
+            persistent congestion |   Slow     |
+        (O)---------------------->|   Start    |
+                                  +------------+
+                                        |
+                                Loss or |
+                        ECN-CE increase |
+                                        v
+ +------------+     Loss or       +------------+
+ | Congestion |  ECN-CE increase  |  Recovery  |
+ | Avoidance  |------------------>|   Period   |
+ +------------+                   +------------+
+           ^                            |
+           |                            |
+           +----------------------------+
+              Acknowledgment of packet
+                sent during recovery
+~~~
+{: #fig-cc-fsm title="Congestion Control States and Transitions"}
+
+These states and the transitions between them are described in subsequent
+sections.
+
+### Slow Start
+
+A NewReno sender is in slow start any time the congestion window is below the
+slow start threshold. A sender begins in slow start because the slow start
+threshold is initialized to an infinite value.
+
+While a sender is in slow start, the congestion window increases by the number
+of bytes acknowledged when each acknowledgment is processed. This results in
 exponential growth of the congestion window.
 
-A sender MUST exit slow start and enter recovery when a loss is detected or when
-the ECN-CE counter reported by the peer increases.
+The sender exits slow start and enters a recovery period when a packet is lost
+or when the ECN-CE count reported by its peer increases.
 
 A sender re-enters slow start any time the congestion window is less than the
-slow start threshold, which only occurs after persistent congestion is declared.
+slow start threshold, which only occurs after persistent congestion is
+declared.
 
-## Congestion Avoidance
+### Recovery Period {#recovery-period}
```

Comment:
I think this should be "Recovery", not recovery period, since "Recovery" is a state, just like Slow Start is a state.  When we only referred to recovery period in a few spots, it seemed ok, but I've always heard the phrase "enter recovery", never "enters a recovery period".